src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -1186,13 +1188,17 @@ private Task WriteAsciiStringAsync(string s)
 
         private async Task WriteStringAsyncSlow(string s)
         {
+            int mask = StaticHttpSettings.EncodingValidationMask;
+
             for (int i = 0; i < s.Length; i++)
             {
-                char c = s[i];
-                if ((c & 0xFF80) != 0)
+                int c = s[i];
+
```

Comment:
NIT: Are all the new lines intentional here?

Reply:
Yes, I like to format code like this, but unsure what is the preferred BCL style.

Reply:
Note: we decided offline to not change this.